R/airtabler.R

```diff
@@ -97,9 +98,47 @@ air_get <- function(base, table_name, record_id = NULL,
         stringsAsFactors =FALSE
       )
   }
+  attr(ret, "offset") <- offset  
   ret
 }
 
+
+#' Get all records from table
+#'
+#' You can retrieve all records from a table. The results will include only records
+#' visible in the order they are displayed.
+#'
+#' @param base Airtable base
+#' @param table_name Table name
+#' @param view (optional) The name or ID of the view
+#' @param sortField (optional) The field name to use for sorting
+#' @param sortDirection (optional) "asc" or "desc". The sort order in which the
+#'   records will be returned. Defaults to asc.
+#' @return A data frame with records
+#' @export
+air_get_all <- function(base, 
+                          table_name,
+                          view = NULL,
+                          sortField = NULL,
+                          sortDirection = NULL) {
```

Comment:
minor indentation issue?

Reply:
Yeah, I fixed it on my rig, but then I did not want to send another commit with this because I invited you for review already. I'll patch it here.